src/overlays/actors/ovl_En_Horse/z_en_horse.c

```diff
@@ -1749,7 +1749,7 @@ void EnHorse_Inactive(EnHorse* this, PlayState* play2) {
             gSaveContext.horseData.scene = play->sceneNum;
 
             // Focus the camera on Epona
-            Camera_SetParam(play->cameraPtrs[CAM_ID_MAIN], 8, this);
+            Camera_SetViewParam(play->cameraPtrs[CAM_ID_MAIN], CAM_VIEW_TARGET, this);
```

Comment:
```suggestion
            Camera_SetViewParam(play->cameraPtrs[CAM_ID_MAIN], CAM_VIEW_TARGET, &this->actor);
```
also in the same file (z_en_horse), in the other `Camera_SetViewParam` call